pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -203,7 +203,7 @@ func createDnsmasqConfiglet(
 		}
 	} else if netconf.Subnet.IP != nil {
 		file.WriteString(fmt.Sprintf("dhcp-option=option:netmask,%s\n",
```

Comment:
I would be good to add a comment here to say that we want the packets to go through the flowlog and ACL code hence we force them to be routed, or something like that.

Reply:
Done

Reply:
Done